docs/mlsys21/v100_throughputs.md

```diff
@@ -0,0 +1,168 @@
+# you will need to provide a placeholder mount point for data directory
+# default is under ${HOME}/datasets
+ubuntu@ip-xxxxxxxx:~/hfta$ bash docker/launch.sh <optional: data directory mount point> <optional: image tag>
+
+=============
+== PyTorch ==
+=============
+
+NVIDIA Release 20.08 (build 15516749)
+PyTorch Version 1.7.0a0+8deb4fe
+
+Container image Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+
+Copyright (c) 2014-2020 Facebook Inc.
+Copyright (c) 2011-2014 Idiap Research Institute (Ronan Collobert)
+Copyright (c) 2012-2014 Deepmind Technologies    (Koray Kavukcuoglu)
+Copyright (c) 2011-2012 NEC Laboratories America (Koray Kavukcuoglu)
+Copyright (c) 2011-2013 NYU                      (Clement Farabet)
+Copyright (c) 2006-2010 NEC Laboratories America (Ronan Collobert, Leon Bottou, Iain Melvin, Jason Weston)
+Copyright (c) 2006      Idiap Research Institute (Samy Bengio)
+Copyright (c) 2001-2004 Idiap Research Institute (Ronan Collobert, Samy Bengio, Johnny Mariethoz)
+Copyright (c) 2015      Google Inc.
+Copyright (c) 2015      Yangqing Jia
+Copyright (c) 2013-2016 The Caffe contributors
+All rights reserved.
+
+Various files include modifications (c) NVIDIA CORPORATION.  All rights reserved.
+NVIDIA modifications are covered by the license terms that apply to the underlying project or file.
+
+NOTE: MOFED driver for multi-node communication was not detected.
+      Multi-node communication performance may be reduced.
+
+root@c7ee88f34a48:/home/ubuntu/hfta#
+
+```
+
+###  Install HFTA and its dependencies as a Python package
+
+```bash
+root@c7ee88f34a48:/home/ubuntu/hfta: pip install -e .
+
+... # additional outputs not shown
+Installing collected packages: pandas, hfta
+  Attempting uninstall: pandas
+    Found existing installation: pandas 1.1.3
+    Uninstalling pandas-1.1.3:
+      Successfully uninstalled pandas-1.1.3
+  Running setup.py develop for hfta
+Successfully installed hfta pandas-1.1.5
+```
+
+
+### Run experiments
+
+1. Prepare datasets
+```bash
+cd /home/ubuntu/hfta
+source datasets/prepare_datasets.sh
+# Download the dataset by calling helper functions defined in `prepare_datasets.sh`. For example: run
+# `prepare_bert` for BERT experiment.
+prepare_bert
+```
+
+2. Prepare experiment workflow helper functions by running benchmarks/workflow.sh
+benchmarks/workflow.sh" expects 3 arguments in total
+- DEVICE: {cuda, tpu}, default is `cuda`
+- DEVICE_MODEL: {a100, v100, etc, or v2/v3 for tpu}, default is `v100`
+- OUTDIR_ROOT: {the output directory of the benchmarked results}, default is `benchmarks`
+
+The first two arguments, are often needed in every run,. Please refer to the script for the usage of the arguments
+```bash
+# The command below will set the target device and device model to be CUDA, A100,
+# and the output directory by default is ./benchmarks, but you can specify other things
+source benchmarks/workflow.sh cuda a100 <optional: output root dir of the benchmark output>
```

Comment:
This is a tutorial for V100 or RTX6000, so the example `source` command should be 
```
source benchmarks/workflow.sh cuda v100
source benchmarks/workflow.sh cuda rtx6000
```